sdk/batch/Azure.Compute.Batch/src/Generated/Docs/BatchClient.xml

```diff
@@ -63,21 +63,7 @@ Uri endpoint = new Uri("<endpoint>");
 TokenCredential credential = new DefaultAzureCredential();
 BatchClient client = new BatchClient(endpoint, credential);
 
-BatchPoolCreateContent pool = new BatchPoolCreateContent("mypool01", "Standard_D1_v2")
-{
-    VirtualMachineConfiguration = new VirtualMachineConfiguration(new ImageReference
-    {
-        Publisher = "MicrosoftWindowsServer",
-        Offer = "WindowsServer",
-        Sku = "2016-datacenter-smalldisk",
-        Version = "latest",
-    }, "batch.node.windows amd64"),
-    TargetDedicatedNodes = 2,
-    NetworkConfiguration = new NetworkConfiguration
-    {
-        EnableAcceleratedNetworking = true,
-    },
-};
+BatchPoolCreateContent pool = new BatchPoolCreateContent(null, null);
```

Comment:
the value here disappears because of a bug introduced in TCGC: https://github.com/Azure/typespec-azure/issues/2175